src/main.js

```diff
@@ -0,0 +1,73 @@
+import './Assets/Styles/style.css';
+const loadComponents = (id, url) =>  {
+    fetch(url)
+    .then((response) => response.text())
+    .then((html) => {
+        document.getElementById(id).innerHTML = html;
+    });
+}
+
+window.onload = () => {
+    loadComponents('header', 'src/Components/header.html');
+    loadComponents('hero', 'src/Components/hero.html');
+    loadComponents('icon-section', 'src/Components/sectionWithIcons.html');
+    loadComponents('maibornWolf', 'src/Components/maibornWolf.html');
+    loadComponents('footer', 'src/Components/footer.html');
+    loadComponents('unsere-teams', 'src/Components/unsere.html');
+    loadComponents('ruanda', 'src/Components/ruanda.html');
+    loadComponents('qualitat', 'src/Components/qualitat.html');
+    loadComponents('managedTeams', 'src/Components/managedTeams.html')
+    loadComponents('kotakt', 'src/Components/kotakt.html');
+    loadComponents('footer', 'src/Components/footer.html');
+    loadComponents('founders', 'src/Components/founderss.html');
+}
+const expandLessMore = (event) => {
+    if (event.target.textContent === 'add'){
+        event.target.textContent = 'remove'
+        event.target.parentNode.nextSibling.nextElementSibling.style['display'] = 'block'
+    }else if (event.target.textContent === 'remove'){
+        event.target.textContent = 'add'
+        event.target.parentNode.nextSibling.nextElementSibling.style['display'] = 'none'
+    }
+    if(event.target.textContent === 'expand_more'){
+        event.target.textContent  = 'expand_less'
+        event.target.parentNode.nextElementSibling.style['display'] = 'block';
+    }else if(event.target.textContent === 'expand_less'){
+        event.target.textContent  = 'expand_more'
+        event.target.parentNode.nextElementSibling.style['display'] = 'none';
+    }
+    else{
+        console.log('Managed Africa team clicked');
+    }
```

Comment:
Here I would not use a lot of nested 'if' or 'else if' statements, as they can make the code hard to read. Since you are checking the conditions individually, you can use a single 'if' statement. Also, please remove the 'console log'; it's only there for testing purposes in the developer tools
```suggestion
const expandLessMore = (event) => {
    if (event.target.textContent === 'add'){
        event.target.textContent = 'remove'
        event.target.parentNode.nextSibling.nextElementSibling.style['display'] = 'block'
    }
    if (event.target.textContent === 'remove'){
        event.target.textContent = 'add'
        event.target.parentNode.nextSibling.nextElementSibling.style['display'] = 'none'
    }
    if(event.target.textContent === 'expand_more'){
        event.target.textContent  = 'expand_less'
        event.target.parentNode.nextElementSibling.style['display'] = 'block';
    }
    if(event.target.textContent === 'expand_less'){
        event.target.textContent  = 'expand_more'
        event.target.parentNode.nextElementSibling.style['display'] = 'none';
    }
    
```